data/aggnd.json

```diff
@@ -3,6 +3,7 @@
     "md5": "357c17f7ee09c8d8d3ef9207adc0671f",
     "pid": "00401653X",
     "bf:Agent": "bf:Organisation",
+    "type": "bf:Organisation",
```

Comment:
Why keeping `bf:Agent` key ? In RERO, we already switch this key (if exists) to `type`, for RERO-ILS usage, this key could be removed

Reply:
To allow a longer transition time we will delete this later with a small fix.